antidote-config.yml

```diff
@@ -11,3 +11,12 @@ vm_config:
   memory: 8192
   cores: 2
   provider: virtualbox
+
+# Providing a value for options supported by the
```

Comment:
I'm not sure it's clear from this comment when I should modify these values. I **think** the idea is that if you're both a) using Vagrant and b) wish to override the default values in selfmedicate.sh, you should uncomment this - but I'd like to see more clarity here if that's true. My over-arching comment about why this only applies to Vagrant still stands though.